BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -3,6 +3,16 @@
 
 class SignalGroup:
     def __init__(self, device: str, name: str, timestamp: str, status_code: str):
+        """
+        Constructor to initialize the signal group
+        # Parameter:
+        # argument1: device id (String)
+        # argument2: device name (String)
+        # argument3: timestamp(String)
+        # argument4: status_code(String)
```

Comment:
Suggestion: One character string representing the initial state of the traffic light (as returned from the API).